addons/mail/static/tests/mock_server/mail_mock_server.js

```diff
@@ -524,8 +519,9 @@ async function discuss_inbox_messages(request) {
     /** @type {import("mock_models").MailMessage} */
     const MailMessage = this.env["mail.message"];
 
-    const { after, around, before, limit = 30, search_term } = await parseRequestParams(request);
+    const { fetch_params = {} } = await parseRequestParams(request);
     const domain = [["needaction", "=", true]];
+    const { search_term, before, after, around, limit = 30 } = fetch_params;
```

Comment:
In new mock server normally you should be able to use `makeKwArgs` to pass `fetch_params` without having to split it.